SU2_CFD/src/solvers/CHeatSolver.cpp

```diff
@@ -667,7 +649,7 @@ void CHeatSolver::Set_Heatflux_Areas(CGeometry *geometry, CConfig *config) {
     }
   }
 
-  SU2_MPI::Allreduce(Local_Surface_Areas, Surface_Areas, config->GetnMarker_HeatFlux(), MPI_DOUBLE, MPI_SUM, SU2_MPI::GetComm());
+  SU2_MPI::Allreduce(Local_Surface_Areas, Surface_Areas.data(), config->GetnMarker_HeatFlux(), MPI_DOUBLE, MPI_SUM, SU2_MPI::GetComm());
```

Comment:
Could this be problematic (MPI + containers)?

Reply:
not that I am aware of :)

Reply:
There is no problem, memory is memory, but when doing this I prefer to use the `.size()` of the container for the MPI count rather than reading it from config or somewhere else.